### ISSUE TITLE
Loudness Sound System files always play at 70hz

### ISSUE THREAD

**Issue by illydan** [adplug/adplug#11]
Loudness Sound System files are fixed to be replayed at 70hz but there are files which need to be replayed at 60hz.
I've ripped the LDS-Files from the German PC Games Magazine - for example: [PCG9707-03.zip](https://github.com/adplug/adplug/files/129585/PCG9707-03.zip)
- those files come in 60hz, 70hz and 85hz versions.

The fix is inside lds.h:
```
-  float getrefresh() { return 70.0f; }
+  float getrefresh() { return 1193182.0/speed; }

```

**Comment by Malvineous** [comment]
Happy to merge the fix, are you willing to send a pull request?

**Comment by illydan** [comment]
Pull request sent - I'm closing this ticket

**Comment by Malvineous** [comment]
Thanks, merged commit a65ef5b
